METIS/headers/FITS_wcu_keywords.yaml

```diff
@@ -0,0 +1,23 @@
+- ext_type: PrimaryHDU
+  keywords:
+
+    HIERARCH:
+      ESO:
+        INS:
+          OPTI17:               # WCU periscope arm
+            ID: "WCU periscope"
+            NAME: "IN"
+          OPTI18:               # WCU PP1 masks
+            ID: "WCU PP1 masks"
+            NAME: "#pupil_masks.current_mask"
+          OPTI19:               # WCU BB exit aperture masks
+            ID: "WCU BB exit aperture"
+            NAME: "#wcu_source.bb_aperture"
+          OPTI20:               # WCU FP2.1 mask wheel
+            ID: "WCU FP2.1 mask"
+            NAME: "#wcu_source.current_fpmask"
+          WCU_LAMP:             # WCU lamp TODO: made up keyword
+            ID: "WCU lamp"
+            NAME: "#wcu_source.current_lamp"
+        SEQ:
+          WCU_BB_TEMP: "#wcu_source.bb_temp"
```

Comment:
It is recommended to use 8 characters for the parameter names in ESO-044156, the Data Interface Control Document. From v7 section 4.4.1.3 "Parameters":

>  In order to keep the size required by the complete hierarchy to a minimum, it is recommended to use names not exceeding 8 characters. 

I'm fine with longer names, but good to do that consciously.

Reply:
Thanks for the link. Yes, it's better to stay consistent with the template manual.